src/AspNetCoreModuleV2/AspNetCore/InProcessShimStaticHtml.htm

```diff
@@ -1,45 +1,83 @@
-                <ul>
-                    <li> The specified version of Microsoft.NetCore.App or Microsoft.AspNetCore.App was not found. </li>
-                    <li> The in process request handler, Microsoft.AspNetCore.Server.IIS, was not referenced in the application. </li>
-                    <li> ANCM could not find dotnet. </li>
-                </ul>
-            </fieldset>
-        </div>
-        <div class="content-container">
-            <fieldset>
-                <h4> Troubleshooting steps: </h4>
-                <ul>
-                    <li> Check the system event log for error messages </li>
-                    <li> Enable logging the application process' stdout messages </li>
-                    <li> Attach a debugger to the application process and inspect </li>
-                </ul>
-            </fieldset>
-            <fieldset>
-                <h4> For more information visit:
-                    <a href="https://go.microsoft.com/fwlink/?LinkID=808681">
-                        <cite> https://go.microsoft.com/fwlink/?LinkID=808681 </cite>
-                    </a>
-                </h4>
-            </fieldset>
-        </div>
-    </div>
-</body>
+<!DOCTYPE html>
+<html lang="en-US" xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8" />
+        <title> HTTP Error 500.0 - ANCM In-Process Startup Failure </title>
+        <style>
+            body {
+                font-family: 'Segoe UI', Tahoma, Arial, Helvetica, sans-serif;
+                font-size: .813em;
+                color: #222;
+                background-color: #fff;
+            }
 
+            h1, h2, h3, h4, h5 {
+                /*font-family: 'Segoe UI',Tahoma,Arial,Helvetica,sans-serif;*/
+                font-weight: 100;
+            }
+
+            h1 {
+                color: #44525e;
+                margin: 15px 0 15px 0;
+            }
+
+            h2 {
+                margin: 10px 5px 0 0;
+            }
+
+            h3 {
+                color: #363636;
+                margin: 5px 5px 0 0;
+            }
+
+            code {
+                font-family: Consolas, "Courier New", courier, monospace;
+            }
+
+            body .titleerror {
+                padding: 3px 3px 6px 3px;
+                display: block;
+                font-size: 1.5em;
+                font-weight: 100;
+            }
+
+            a {
+                color: #1ba1e2;
+                text-decoration: none;
+            }
+
+                a:hover {
+                    color: #13709e;
+                    text-decoration: underline;
+                }
+
+            li {
+                margin: 5px;
+            }
+
+        </style>
+    </head>
+    <body>
+        <h1> HTTP Error 500.0 - ANCM In-Process Startup Failure </h1>
+
+        <h2> Common causes of this issue: </h2>
+
+        <ul>
+            <li> The specified version of Microsoft.NetCore.App or Microsoft.AspNetCore.App was not found. </li>
```

Comment:
@shirhatti please review the list of recommendations here (for language/effectiveness)